src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -413,7 +413,7 @@ private void OnNodeViewMouseEnter(object sender, MouseEventArgs e)
             // Always set old ZIndex to the last value, even if mouse is not over the node.
             oldZIndex = NodeViewModel.StaticZIndex;
 
-            if (!previewEnabled) return; // Preview is hidden. There is no need run further.
+            if (!previewEnabled || !ViewModel.IsPreviewInsetVisible) return; // Preview is hidden. There is no need run further.
```

Comment:
@Racel , If you are asking about http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-9737 then no.
Crash in http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-9737 is because `Data` in `MirrorData` is null, it's not somehow connected with this change.